src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
```

Comment:
2. **Unused Private Method**:
    - The private method `unusedPrivateMethod()` is not used anywhere in the class. It should be removed to avoid clutter and improve readability.